wiki-config

```diff
@@ -1,16 +1,38 @@
-# wiki-config
+#!/bin/bash
+
+#set -x
+set -e
+set -o pipefail
+set -o nounset
```

Comment:
this sort of stuff should be in the scripts, not in the config file

Reply:
Was me being lazy.